packages/waku/src/minimal/client.ts

```diff
@@ -29,7 +30,7 @@ const DEFAULT_HTML_HEAD = [
   createElement('meta', { name: 'generator', content: 'Waku' }),
 ];
 
-const BASE_RSC_PATH = `${import.meta.env?.WAKU_CONFIG_BASE_PATH}${
+const BASE_RSC_PATH = `${import.meta.env?.WAKU_CONFIG_BASE_PATH && withTrialSlash(import.meta.env?.WAKU_CONFIG_BASE_PATH)}${
```

Comment:
If any easier, we can normalize and ensure trailing slash here, so runtime don't need to worry about it

https://github.com/wakujs/waku/blob/4585faa63756180beb1998550ae0a5257da6ff04/packages/waku/src/lib/vite-rsc/plugin.ts#L106-L108